web/public/mgmt/config.yml

```diff
@@ -2346,9 +2346,24 @@ collections:
             collapsed: false
             widget: object
             fields:
-              - { label: Violation Notice Message Text, name: violationNoticeMessage, widget: string }
-              - { label: Violation Notice CTA Text, name: violationNoticeCTA, widget: string }
-              - { label: Violation Notice CTA Link, name: violationNoticeCTALink, widget: no-space }
+              - {
+                  label: Violation Notice Message Text,
+                  name: violationNoticeMessage,
+                  widget: string,
+                  required: false,
+                }
+              - {
+                  label: Violation Notice CTA Text,
+                  name: violationNoticeCTA,
+                  widget: string,
+                  required: false,
+                }
+              - {
+                  label: Violation Notice CTA Link,
+                  name: violationNoticeCTALink,
+                  widget: no-space,
+                  required: false,
+                }
```

Comment:
These fields aren't used in the CMS despite being required (like no corresponding JSON object exist for them).

I wanted @SirSamuelJoseph to clarify whether this should be deleted or modified. Since this blocks Kellie I'm going to make these optional in the interim